go-controller/pkg/ovn/policy_common.go

```diff
@@ -22,6 +22,7 @@ type namespacePolicy struct {
 	namespace       string
 	ingressPolicies []*gressPolicy
 	egressPolicies  []*gressPolicy
+	podIPMap        map[string]map[string]string
```

Comment:
maybe add a comment explaining what that means exactly? (ie, what each string is)

although wait, it seems like you don't actually use this at all? You add stuff to it and delete from it but you never look at it other than that